nrf52-hal-common/src/clocks.rs

```diff
@@ -33,7 +33,7 @@ pub struct Clocks<H, L, LSTAT> {
     periph: CLOCK,
 }
 
-impl<H, L, LSTAT> Clocks<H, L, LSTAT> {
+impl Clocks<Internal, Internal, LfOscStopped> {
     pub fn new(clock: CLOCK) -> Clocks<Internal, Internal, LfOscStopped> {
```

Comment:
For future reference: If the type arguments is fully specified in the `impl` declaration, the return type of the constructor can be `Self`, which reduces duplication.